README.md

```diff
@@ -4,6 +4,8 @@
 ## About this repository
 
 This repo is the canonical source for Kubernetes Operators that appear on [OperatorHub.io](https://operatorhub.io).
+The solutions merged on this repository are distributed via the [OLM][olm] index catalog [quay.io/operatorhubio/catalog][quay.io].
+Users can install [OLM][olm] in any Kubernetes or vendor such as Openshift to consume this content. [(more info)][catalog]
```

Comment:
I think, there is no need to describe manual OLM install, it is part of install output by hitting `Install`. Try it yourself https://operatorhub.io/operator/akka-cluster-operator

Reply:
Any person can use the operators directly and without OLM. However, we built the catalogue/image to be used with OLM right? So, it seems to me that we need to describe that. So, the added info is specific to the image built and how it is used/useful for users.